elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/query/QueryTranslator.java

```diff
@@ -132,7 +134,7 @@ public QueryTranslator(SQLReferenceTable referenceTable, SQLDialect sqlDialect)
         String tableAlias = applyQuotes(table.getAlias());
 
         String tableStatement = tableCls.isAnnotationPresent(FromSubquery.class)
-                ? "(" + tableCls.getAnnotation(FromSubquery.class).sql() + ")"
+                ? "(" + context.resolve(tableCls.getAnnotation(FromSubquery.class).sql()) + ")"
```

Comment:
Construct the table context here from the queryable.  It is not referenced anywhere else.